_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -18,141 +18,123 @@ class AnnualIncomeSection extends React.Component {
 
     return (
       <div>
-        <div className="row">
-          <div className="small-12 columns">
-            <h4>Annual Income</h4>
-            {this.props.external.receivesVaPension === true &&
-              <p>
-                <strong>
-                You are not required to enter financial information because you
-                indicated you are receiving a VA pension.
-                </strong>
-              </p>
-            }
-          </div>
+        <h4>Annual Income</h4>
+        {this.props.external.receivesVaPension === true &&
```

Comment:
We should move this out so the conditional span is created outside of the return() statement and put in a variable.  This sort of conditional logic with a short-circuit and works, but it's a bit hard to read to me as it's too many lines.

I would prefer pulling the static content out into a variable which just gets default initialized to null or an empty string.